### DISCUSSION TITLE
Getting migration failed error after upgrading to 14.0

### DISCUSSION THREAD

**Discussion by heyarviind** [pocketbase/pocketbase#2180] [Q&A] [answered]
I downloaded 14.0 and when I restarted the server it gave me an error, can someone help to fix this, please?

![Screenshot 2023-03-30 at 2 57 34 PM](https://user-images.githubusercontent.com/21043614/228792767-9702b3f1-70bc-41ca-80d8-4096136950de.png)

**Comment by ganigeorgiev** [comment, marked as answer]
This is most likely because you have records with duplicated unique field values. It could happened if the unique field option was enabled after the duplicated records were inserted and since now the unique check is handled on DB level it will fail the constraint.

You can try to normalize first your `categories.color` values and then upgrade.

**Reply by heyarviind** [reply]
worked thanks :)
